varcode/variant.py

```diff
@@ -50,18 +50,37 @@
     ExonLoss,
     ExonicSpliceSite,
 )
-from .nucleotides import normalize_nucleotide_string
+from .nucleotides import normalize_nucleotide_string, STANDARD_NUCLEOTIDES
 from .string_helpers import trim_shared_flanking_strings
 from .transcript_helpers import interval_offset_on_transcript
 
 class Variant(object):
+    __slots__ = (
+        "contig",
+        "start",
+        "end",
+        "ref",
+        "alt",
+        "ensembl",
+        "original_ref",
+        "original_alt",
+        "original_start",
+        "_short_description",
```

Comment:
This looks like a maintenance nightmare, does it substantially improve performance? Or, were you running into memory usage problems?